docs/src/guides/tables/merge_insert.md

```diff
@@ -71,6 +71,45 @@ with merge insert, enable both `when_matched_update_all()` and
     If a column is nullable, it can be omitted from input data and it will be
     considered `null`. Columns can also be provided in any order.
 
+### Conditional Updates
```

Comment:
shouldn't the doc change go to the new https://github.com/lancedb/documentation?

Reply:
I've been waiting for that website to be ready. Last I heard it's being totally redone again to be using Hugo. The current published version of that lacks basic features like a search bar, so I don't think our users will find it helpful until it's fixed.